requirements.txt

```diff
@@ -1,4 +1,4 @@
-Cython>=0.22
```

Comment:
@AlexJoz 
Is this file still needed? The requirements are already listed ipy `pyproject.toml`.

Reply:
@ibayer used in previous gitlab/docker setup. Probably we can export reqs from pyproject file on build step, but not related. Some sudden spaces.. don't need this file in current mr